src/main/java/jenkins/plugins/logstash/remoteLogging/RemoteLogstashWriter.java

```diff
@@ -69,6 +69,7 @@ public RemoteLogstashWriter(Run run, Jenkins jenkins) {
 
         this.jenkinsUrl = jenkins.getRootUrl();
         this.buildData = new BuildData(run, new Date());;
+        init();
```

Comment:
This was not set up to do logging from the master, which for Pipeline I need: even if the entire build is wrapped in a (remote) `node`, log messages about control flow are still sent from the master.